notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -83,7 +83,9 @@
     "import hera_pspec as hp\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    from HERA_FRF_cov import FRF_cov_calc
</pre><p>I don't think this is a HERA team repo, and we would like to avoid dependencies outside HERA team (except big public repos, of course). Do we need it? And if so, can we get this code in a HERA repo (or in a new one owned by hera_team) with all the normal unit testing and coverage, etc.? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
We've worked out on the pspec call that this code will live in `hera_cal`.